types/pdfmake/interfaces.d.ts

```diff
@@ -31,11 +31,13 @@ export interface TFontDictionary {
     [fontName: string]: TFontFamilyTypes;
 }
 
+export type FontType = string | Buffer | Uint8Array | ArrayBuffer;
```

Comment:
Since the types from PDFKit are referenced here, you could directly reference the PDFFontSource type from PDFKit. I think it would be a better approach since that's behaviour pdfMake merely inherits from PDFKit.

Reply:
Right but that's true for pretty much all of pdfmake - and so while I agree it's a good change, it should probably be made in another PR

Reply:
> The types that directly correspond to types defined by PDFKit are already imported from PDFKit

Ah I missed that sorry